content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -91,47 +107,68 @@ This race condition is mitigated by a "detect and repair" method.
+
+The Istio CNI and Istio discovery chart use different values that require you set the following, either in an overrides values file or at your command prompt when installing the `istiod` chart, to manage network annotations when chaining CNI plugins:
+
+* `values.istio_cni.enabled` should be set to the same value as `values.cni.enabled`.
+
+* `values.istio_cni.chained` should be set to the same value as `values.cni.chained`.
+
+{{< text bash >}}
+$  helm install istiod istio/istiod -n istio-system --set values.istio_cni.enabled=true --wait
+{{< /text >}}
+
 ### Hosted Kubernetes settings
 
 The `istio-cni` plugin is expected to work with any hosted Kubernetes version using CNI plugins.
 The default installation configuration works with most platforms.
 Some platforms required special installation settings.
 
-* Google Kubernetes Engine
-
-    {{< text yaml >}}
-    apiVersion: install.istio.io/v1alpha1
-    kind: IstioOperator
-    spec:
-      components:
-        cni:
-          enabled: true
-          namespace: kube-system
-      values:
-        cni:
-          cniBinDir: /home/kubernetes/bin
-    {{< /text >}}
-
-* Red Hat OpenShift 4.2+
-
-    {{< text yaml >}}
-    apiVersion: install.istio.io/v1alpha1
-    kind: IstioOperator
-    spec:
-      components:
-        cni:
-          enabled: true
-          namespace: kube-system
-      values:
-        sidecarInjectorWebhook:
-          injectedAnnotations:
-            k8s.v1.cni.cncf.io/networks: istio-cni
-        cni:
-          cniBinDir: /var/lib/cni/bin
-          cniConfDir: /etc/cni/multus/net.d
-          cniConfFileName: istio-cni.conf
-          chained: false
-    {{< /text >}}
+{{< tabset category-name="cni-platform" >}}
+
+{{< tab name="Google Kubernetes Engine" category-value="gke" >}}
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  components:
+    cni:
+      enabled: true
+      namespace: kube-system
+  values:
+    cni:
+      cniBinDir: /home/kubernetes/bin
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="Red Hat OpenShift 4.2+" category-value="ocp" >}}
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  components:
+    cni:
+      enabled: true
+      namespace: kube-system
+  values:
+    cni:
+      cniBinDir: /var/lib/cni/bin
+      cniConfDir: /etc/cni/multus/net.d
+      cniConfFileName: istio-cni.conf
+      chained: false
+{{< /text >}}
+
+{{< tip >}}
+Note: Using `sidecarInjectorWebhook` to inject the `k8s.v1.cni.cncf.io/networks: istio-cni` annotation will overwrite preexisting annotations of the same name. The sidecar injection templates will automatically append the `istio-cni` network value to the `k8s.v1.cni.cncf.io/networks` annotation on pods if the following values are properly set for OpenShift: `istio_cni.enabled=true` and `istio_cni.chained=false`. If no network annotation currently exists, it will automatically create one for `istio-cni`.
```

Comment:
TBH I am not sure this is needed... it seems useful now, but I feel like we should be saying what you _should_ do, not what you should not do.

Given we don't document `k8s.v1.cni.cncf.io/networks`, no users _reading this doc_ will ever set it. Instead, they will see this note and be confused.

Where this could be useful is a troubleshooting doc -- someone with `k8s.v1.cni.cncf.io/networks` has broken configuration. But even then, its completely broken, so not sure how they would ever get into this state?

Reply:
I can take that out if we don't think it's useful.  My thought was that if someone was using the webhook injection to add the annotation that the configuration could cause issues.